.github/workflows/update-snapcraft.yaml

```diff
@@ -0,0 +1,64 @@
+on:
+  workflow_call:
+    inputs:
+      openstack-release:
+        description: OpenStack release
+        type: string
+        required: true
+
+env:
+  SNAP_REPO_PATH: snap-tempest
+  AUTOMATION_REPO_PATH: snap-tempest-automation
+
+jobs:
+  update-snapcraft:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout snap-tempest
+        uses: actions/checkout@v4
+        with:
+          repository: canonical/snap-tempest
+          ref: stable/${{ inputs.openstack-release }}
+          path: ${{ env.SNAP_REPO_PATH }}
+      - name: Checkout snap-tempest-automation
+        uses: actions/checkout@v4
+        with:
+          ref: main
+          path: ${{ env.AUTOMATION_REPO_PATH }}
+      - name: Install tox
+        run: |
+          sudo apt-get update
+          sudo apt-get install --yes tox
+      - name: Update Snapcraft YAML
+        id: update_snapcraft
+        run: |
+          tox -c ${{ env.AUTOMATION_REPO_PATH }}/tox.ini -e release -- \
+              --input ${{ env.SNAP_REPO_PATH }}/snap/snapcraft.yaml \
+              --output ${{ env.SNAP_REPO_PATH }}/snap/snapcraft.yaml \
+              --release ${{ inputs.openstack-release }} \
+              --manual-requirements ${{ env.SNAP_REPO_PATH }}/requirements-manual.txt \
+              --excluded-plugins ${{ env.SNAP_REPO_PATH }}/excluded-plugins.txt
```

Comment:
These envs should now be `env.AUTOMATION_REPO_PATH`?

Reply:
no, it's meant to be that way, to allow us to set per-OpenStack-release manual requirements and excluded plugins. If those paths don't exist, the default kicks in and the ones from AUTOMATION_REPO_PATH are taken

Reply:
I see thanks for the clarification!